src/Sil/Component/Contact/Model/Contact.php

```diff
@@ -248,4 +303,59 @@ public function getPhones(): Collection
     {
         return $this->phones;
     }
+
+    /**
+     * Add a Phone to the collection.
+     *
+     * @param PhoneInterface $phone
+     */
+    public function addPhone(PhoneInterface $phone)
+    {
+        if ($this->phones->contains($phone)) {
+            throw new \InvalidArgumentException('This contact already owns this phone');
+        }
+
+        $this->phones->add($phone);
+
+        if (!$this->defaultPhone) {
+            $this->setDefaultPhone($phone);
+        }
+    }
+
+    /**
+     *  Remove a Phone from the collection.
+     *
+     * @param PhoneInterface $phone
+     */
+    public function removePhone(PhoneInterface $phone)
```

Comment:
Remarks related to `public function removeAddress(AddressInterface $address)`'s comment